In guile 2.0, testsuite fails with

In file included from ../../libguile/scmconfig.h:27,
                 from /home/janneke/vc/gub/target/tools/src/guile-2.0.0/libguile/__scm.h:54,
                 from /home/janneke/vc/gub/target/tools/src/guile-2.0.0/libguile.h:31,
                 from /home/janneke/vc/gub/target/tools/src/guile-2.0.0/test-suite/standalone/test-asmobs-lib.c:23:
../../lib/time.h:469: error: expected ';', ',' or ')' before '__timer'
../../lib/time.h:491: error: expected ';', ',' or ')' before '__timer'


$ diff -purN guile-2.0.0.1/Makefile.am~ guile-2.0.0.1/Makefile.am
--- guile-2.0.0.1/Makefile.am~	2011-03-16 19:14:29.724924881 +0100
+++ guile-2.0.0.1/Makefile.am	2011-03-16 19:15:17.970123750 +0100
@@ -33,8 +33,6 @@ SUBDIRS =					\
 	guile-readline				\
 	examples				\
 	emacs					\
-	test-suite				\
-	benchmark-suite				\
 	gc-benchmarks				\
 	am					\
 	doc
[1]19:15:35 janneke@vuurvlieg:~/vc/gub/target/tools/src
